bin/eslint.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+'use strict';
+
+var client = require('../lib/client');
+var useStdIn = (process.argv.indexOf('--stdin') > -1);
```

Comment:
The stdin option is already supported. This file could simply `require('./eslint_d.js');`.